qcengine/programs/tests/test_programs.py

```diff
@@ -200,3 +200,17 @@ def test_random_failure_with_success(failure_engine):
     assert ret.success, ret.error.error_message
     assert ret.provenance.retries == 1
     assert ret.extras["ncalls"] == 2
+
+@testing.using_openmm
+def test_openmm_task():
+    json_data = copy.deepcopy(_base_json)
+    json_data["molecule"] = qcng.get_molecule("water")
+    json_data["driver"] = "energy"
+    json_data["model"] = {"method": "smirnoff-1.1.0",
+                          "url": None,
+                          "offxml": "test_forcefields/smirnoff99Frosst.offxml"}
+    json_data["keywords"] = {}
+
+    ret = qcng.compute(json_data, "openmm", raise_error=True)
+
+    assert ret.success is True
```

Comment:
Since the units are in question here could you add a test to `test_procedures::179` which does a good job of checking these values?

Reply:
@dgasmith I'm not sure what you mean here. Can you clarify?

Reply:
There are geometry optimization tests in `test_procedures`, it would be good to add an OpenMM one there to help verify units.

Reply:
We have decided that geometry optimization tests for OpenMM are out of scope for this PR, but should be created as a separate PR.